packages/api-headless-cms/src/handler/plugins/graphqlFields/richText.ts

```diff
@@ -0,0 +1,66 @@
+
+import { CmsModelFieldToGraphQLPlugin } from "@webiny/api-headless-cms/types";
+
+const createListFilters = ({ field }) => {
+    return `
+        # Matches if the field is equal to the given value
+        ${field.fieldId}: JSON
+
+        # Matches if the field is not equal to the given value
+        ${field.fieldId}_not: JSON
+
+
+        # Matches if the field value equal one of the given values
+        ${field.fieldId}_in: [JSON]
+
+        # Matches if the field value does not equal any of the given values
+        ${field.fieldId}_not_in: [JSON]
+
+        # Matches if given value is a substring of the the field value
+        ${field.fieldId}_contains: JSON
+
+        # Matches if given value is not a substring of the the field value
+        ${field.fieldId}_not_contains: JSON
+    `;
+};
+
+const plugin: CmsModelFieldToGraphQLPlugin = {
+    name: "cms-model-field-to-graphql-rich-text",
+    type: "cms-model-field-to-graphql",
+    fieldType: "rich-text",
+    isSortable: true,
```

Comment:
`isSortable: true` - remove this. Also while you're at it, search for all "sortable" strings in `api-headless-cms`, and remove those. Check the tests as well. there'll probably be some breaking, we'll need to adjust them to this change.

If you'll need help, just let me know.